bin/segment/normalizefile.py

```diff
@@ -76,9 +76,12 @@ def runMakeMethodRef(species, sampleList, methodName, lowessDir):
 		return [False]
 		
 	binArray = common.importInfoFile(normVars.binDict[species], [0, 1, 2, 4, 6], 'normref', skiprows=0)
-	xLocs = [x for x,y in enumerate(binArray['chrom']) if y == 'chrX']
-	yLocs = [x for x,y in enumerate(binArray['chrom']) if y == 'chrY']
-	
+	if species == "hg38":
```

Comment:
```suggestion
	assert species in ["hg38", "mm10"], "Unrecognized species!!!"
	if species == "hg38":
```

So, as these lines read here, the code will ONLY work if species is hg38 or mm10. How many other places in the code is this true? Do you think there will ever be a need to run other species (or genome versions)? If the answer to the first question is "multiple" and/or the answer to the second one is "no", then this is fine -- but we need a note about this limitation in the README. 

Otherwise, this could...probably...be set to look for the last two unique values for chromosomes -- something like:
xchrom = np.unique(binArray['chrom'])[-2]
ychrom = np.unique(binArray['chrom'])[-1]
xLocs = [x for x,y in enumerate(binArray['chrom']) if y == xchrom]
yLocs = [x for x,y in enumerate(binArray['chrom']) if y == ychrom]

What do you think is safer/preferable?

Reply:
I think adding the assert statement to ensure that only hg38 and mm10 genomes are used is probably the easiest safe fix. 
Anyone that would like to switch to a different species or genome build would have to edit the config and argument files anyway as well because those are only configured for hg38 and mm10 at this point anyway, so I think if we add a note in the README that this is something that would need to be changed as well if an unsupported genome is used it should be ok. 
Though, I guess the argument could also be made that the fewer places in the code that someone has to make changes, the better, in which case, we could implement your second suggestion and just make a note that in the genome we are assuming that the second to last chromosome is X and the second to last chromosome is Y.

Reply:
In that case, my leaning would be to assume the last two chromosomes are X and Y, and to add a note in the main README that you must have all chromosomes including X/Y in your reference (but none of those weird haplotype/unassembled contigs).

Reply:
ok! I will make those edits and add a note in the main README about the last two chromosomes/alternate chrs